CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 4.0.0 (2019-09-13)
 
+* **colors:** Add Mozilla brand colors
```

Comment:
This should be added to the change log under a `#HEAD` and not as part of the existing 4.0 version.